File: webapi/Models/Usuario.cs
using System;
using System.Collections.Generic;

namespace webapi.Models
{
    public class Usuario
    {
        public int Id { get; set; }
        public string  Nombre { get; set; }
        public byte[] PasswordHash { get; set; }
        public byte[] PasswordSalt { get; set; }
        public string Genero { get; set; }

        public DateTime FechaNacimiento { get; set; }
        public string  Alias { get; set; }
        public DateTime Creado { get; set; }
        public DateTime UltimaConexion { get; set; }
        public string Introduccion { get; set; }
        public string BuscarPor { get; set; }
        public string Intereses { get; set; }
        public string  City { get; set; }
        public string Pais { get; set; }
        public ICollection<Foto> Fotos { get; set; }
        public ICollection<Like> Likers { get; set; }
        public ICollection<Like> Likees { get; set; }

    }
}

File: SPA/src/app/model/Usuario.ts
import { Foto } from './Foto';

export interface Usuario  {
    id: number;
    nombre: string;
    genero: string;
    edad: number;
    alias: string;
    creado: Date;
     ultimaConexion: Date;
     introduccion?: string;
     buscarPor?: string;
     intereses?: string;
      city: string;
     pais: string;
     url: string;
     fotos: Foto[];
}
